test/pages/assets.t/run.t

```diff
@@ -0,0 +1,91 @@
+Blablabla
+
+  $ cat index.mld
+  {0 Package page}
+  
+  Some image:
+  {%html: <img src="img.jpg" />%}
+
+And we'll have a module that we'll put underneath this package page.
+
+  $ cat test.mli 
+  (** Humpf, let's try accessing the asset:
+    {%html: <img src="../img.jpg" />%}
+    *)
+  
+  (** Nevermind *)
+  type t
+  
+
+Compile the module first
+
+  $ ocamlc -c -bin-annot test.mli
+
+Then we need to odoc-compile the package mld file, listing its children
+
+  $ odoc compile index.mld --child module-test --child asset-img.jpg
+
+This will have produced a file called 'page-index.odoc'.
+Now we can odoc-compile the module odoc file passing that file as parent.
+
+  $ odoc compile test.cmti -I . --parent index
+
+Link and generate the HTML (forgetting the asset!):
+
+  $ for i in *.odoc; do odoc link -I . $i; done
+  $ for i in *.odocl; do odoc html-generate $i -o html; done
+  File "img.jpg":
+  Warning: asset is missing.
```

Comment:
I wondered if this should have been a fatal error, since I didn't know how to make it be one I managed to convince myself that it shouldn't be.

Reply:
I think `odoc` tries to avoid failing too badly in case of errors.

So I don't think it should be a hard failure, but that should not stop processing the other assets (see above)